woudc_data_registry/models.py

```diff
@@ -818,7 +841,7 @@ def unpack_station_names(rows):
         if name.startswith('\\x'):
             name = decode_hex(name[2:])[0].decode('utf-8')
             row['name'] = name
-            row['identifier'] = ':'.join([station, name])
+            row['station_name_id'] = ':'.join([station, name])
```

Comment:
Turns out neither the `'identifier'` or `'station_name_id'` keys are used, so this line can be deleted.